src/govuk/components/checkboxes/template.njk

```diff
@@ -88,7 +88,7 @@
           {% if hasHint %}
           {{ govukHint({
             id: itemHintId,
-            classes: 'govuk-checkboxes__hint',
+            classes: 'govuk-checkboxes__hint' + (' ' + item.hint.classes if item.hint.classes),
```

Comment:
Non-blocking I wonder if we could avoid the extra whitespace by doing something like

`'govuk-checkboxes__hint' + ((' ' + item.hint.classes) if item.hint.classes)`

Hopefully this'd output the space only when there are classes.

Reply:
This doesnt add whitespace after all so works well.